docs/docs/15-concepts.md

```diff
@@ -33,9 +37,10 @@ more "prod" stages at the end.
 
 ### What is Freight?
 
-**Freight** is Kargo's second most important concept. A single "piece of
-freight" is a set of references to one or more versioned artifacts, which may
-include one or more:
+**Freight** is Kargo's second most important concept. Frieight generally refers
+to a collection of artifacts that gets promoted together to the next stage.
+A single "piece of freight" is a set of references to one or more versioned
+artifacts, which may include one or more:
```

Comment:
tbh, I feel this change adds more confusion. We should be careful of unnecessarily using words like "generally," as it implies there are exceptions to what you're about to say. Freight isn't _generally_ a collection of artifacts. Freight _is_ a collection of artifacts. There isn't an exception to that.

The bit about Freight being what moves from stage to stage is already stated just a few lines down from here.

imho, I would revert this change.

```suggestion
**Freight** is Kargo's second most important concept. A single "piece of
freight" is a set of references to one or more versioned artifacts, which may
include one or more:
```